jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/saphana/SapHanaSqlDialect.java

```diff
@@ -0,0 +1,143 @@
+package com.exasol.adapter.dialects.saphana;
+
+import static com.exasol.adapter.AdapterProperties.*;
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.*;
+import static com.exasol.adapter.capabilities.AggregateFunctionCapability.VAR_SAMP;
+import static com.exasol.adapter.capabilities.LiteralCapability.*;
+import static com.exasol.adapter.capabilities.MainCapability.*;
+import static com.exasol.adapter.capabilities.PredicateCapability.*;
+import static com.exasol.adapter.capabilities.ScalarFunctionCapability.*;
+
+import java.sql.*;
+import java.util.*;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.capabilities.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+import com.exasol.adapter.sql.*;
+
+/**
+ * This class implement the SAP HANA SQL dialect.
+ *
+ * @see <a href=
+ *      "https://help.sap.com/viewer/4fe29514fd584807ac9f2a04f6754767/2.0.03/en-US/f327b70cae564c53a766367a8aad0164.html">SAP
```

Comment:
This link is portal-user-specific. Please replace by either a publicly viewable link or an abstract description of how to view the HANA SQL reference.

Reply:
replace the link. Please, try it out